config/AcConstants.ini

```diff
@@ -3,10 +3,21 @@
 [Component]
 
 # Define numbers of ports 
-
 ActiveRateGroupOutputPorts          =       10  ; Number of rate group member output ports for ActiveRateGroup
 CmdDispatcherComponentCommandPorts  =       30  ; Used for command and registration ports 
 CmdDispatcherSequencePorts          =       5   ; Used for uplink/sequencer buffer/response ports
 RateGroupDriverRateGroupPorts       =       3   ; Used to drive rate groups
 HealthPingPorts                     =       25  ; Used to ping active components
 FileDownCompletePorts               =       1   ; Used for broadcasting completed file downlinks
+
+; Hub connections. Connections on all deployments should mirror these settings.
+GenericHubInputPorts = 10
+GenericHubOutputPorts = 10
+GenericHubInputBuffers = 10
+GenericHubOutputBuffers = 10
+
+; Outputs from the generic repeater.
+GenericRepeaterOutputPorts = 2
+
+
+StaticMemoryAllocations = 4
```

Comment:
Optional: These are some bonus config options that aren't used in this PR (but will be used in future revisions), we might want to remove them from this PR

Reply:
I am going to leave it.  They will be needed soon, and I don't want to try pulling apart individual files.